css/secondary-page.styl

```diff
@@ -62,6 +62,7 @@
               background: rgba(255,255,255, 0.2)
               color: rgba(255,255,255,1)
             &.active
+            &[aria-selected=true]
```

Comment:
Should these also be styled to match the top nav active states? (Different padding, no bold)